contrib/systemd/lxa-iobus.service

```diff
@@ -12,4 +12,4 @@ RestartSec=30
 
 
 [Install]
-WantedBy=netowrk.target
+WantedBy=network.target
```

Comment:
While `netowrk.target` is obviously a typo I am not too sure if `network.target` is actually the right systemd unit to be `WantedBy=`.
On my Arch Linux install (which I would consider a good reference when it comes to clean systemd integration) both `nginx.service` and `sshd.service` (as examples for simple daemons) specify `WantedBy=multi-user.target` in the `[Install]` section.

Specifying `After=network.target` in `[Unit]` should however be the right thing to do.

We should adopt whatever is the best practice.

Reply:
I am not quite sure what is correct here. On our production systems the service works fine, even with the typo in the ``After=`` statement.

With ``RestartSec=30`` it seems to be not that important on what we depend :-)

Reply:
It looks to me like we do not have an `After=` statement at all, but maybe we should.
I would suggest adding `After=network.target` to the `[Unit]` section (so we are only started once network interfaces are available (they may however not be up yet, for that we would AFAIK need `After=network-online.target`)).
And changing the `WantedBy=network.target` to `multi-user.target`.

Reply:
https://github.com/linux-automation/lxa-iobus/pull/24 is an alternative to this PR. Please take a look there.